File: C++ Programming/problems on numberical/problem91.cpp
/*
91. Display ASCII value of given character
	
	Author :Milind Nalawade
	
	Date : 13/07/2021
	
	Language : C++ programming
	
*/
using namespace std;
#include<iostream>

void DisplayASCII(char ch)
{
	cout<<"\n The ASCII value of the "<<ch" is = "<<int(ch);
}

int main()
{
	char ch='\0';
	cout<<"\n Enter your character : ";
	cin>>ch;
	DisplayASCII(ch);
	return 0;
}	

File: Java programming/problems on numbericals/problem91.java
/*

91. Display ASCII value of given character
	
	Author :Milind Nalawade
	
	Date : 13/07/2021
	
	Language : Java 14.0.2
	
*/
import java.lang.*;
import java.util.*;

class Logic 
{
	public static void DisplayASCII(char ch)
	{
		int ascii1=ch;
		System.out.println("\n The ASCII value of "+ch+" is ="+ascii1);
	}
}

class problem91 extends Logic
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.print("\n Enter your character : ");
		char ch=sobj.next().charAt(0);
		DisplayASCII(ch);
	}	
}